docs/source/guide/predictions.md

```diff
@@ -8,9 +8,9 @@ meta_description: Label Studio Documentation for importing predicted labels, pre
 
 If you have predictions generated for your dataset from a model, either as pre-annotated tasks or pre-labeled tasks, you can import the predictions with your dataset into Label Studio for review and correction. Label Studio automatically displays the pre-annotations that you import on the Labeling page for each task. 
 
-To import predicted labels into Label Studio, you must use the [Basic Label Studio JSON format](tasks.html#Basic-Label-Studio-JSON-format) and set up your tasks with the `predictions` JSON key. The Label Studio ML backend also outputs tasks in this format. 
+To import predicted labels into Label Studio, you must use the [Basic Label Studio JSON format](tasks.html#Basic-Label-Studio-JSON-format) and set up your tasks with the `predictions` JSON key. The Label Studio ML backend also outputs tasks in this format. You must use different IDs for each task elements, annotations, predictions and their `result` items. 
```

Comment:
actually, you don't need to specify IDs when importing tasks with preannotations.

Reply:
Ah okay cool, this was a holdover from the previous docs

Reply:
Removed